### ISSUE TITLE
Color to Contrast ratio on Popup Balloons

### ISSUE THREAD

**Issue by alkaris2** [flameshot-org/flameshot#1496]
**Flameshot version**
v0.9.0
Compiled with Qt 5.15.2

**Describe the bug**

The color contrast ratio of these pop-up balloons are unreadable. White text on Yellow background is not readable.

![image](https://user-images.githubusercontent.com/73155407/111854168-c84f1a80-896d-11eb-80bb-a80642f9dd27.png)

I keep seeing other popup balloons popup with long text, and it's the same white text on yellow background, which is 100% unreadable. A fix needs to be made so the pop-up balloons follow systems themes or hardcode a different kind color for the pop-up balloons, or instead dump pop-up balloons entirely and just use native OSD messages for all message notifications from the utility.

**System Information**
Manjaro Linux

**Comment by borgmanJeremy** [comment]
You can disable the pop up in the config.

**Comment by borgmanJeremy** [comment]
This is also entirely dependent on your DE which you did not post any details on. We do in fact use native dialogs when available. For instance on KDE this shows up as any other notification would. Same for Windows / Macos.

**Comment by alkaris2** [comment]
I have turned them off in the config, but the popup balloons still appear. sooo that needs to be fixed, because it's clearly not reading the options that I've set.

**Comment by mmahmoudian** [comment]
@alkaris2 I turned off my notification and it is properly turned off. Would you please post your system information (OS, Desktop Environment, Flameshot version, ...) along with your Flameshot config (you can use the Export button at the bottom of the Configuration window:

![image](https://user-images.githubusercontent.com/390889/112299321-5bef5680-8ca0-11eb-847c-90101e909462.png)
